### PR TITLE
Document release process

### PR DESCRIPTION
**Pull request by vmttn**
## Description
Self explanatory :)

---
This change is [Reviewable](https://reviewable.io/reviews/arkhn/warehouse-api/157)

### CODE REVIEW

File: fhir-api/CONTRIBUTING.md [hunk 1]
```diff
@@ -59,3 +59,38 @@ Code quality is enforced with `pre-commit` hooks: `black`, `isort`, `flake8`
...
+
+## Builds
+
+Each push (commits and/or tags) will publish a single image to the DockerHub registry.
+
+Each image will have one or more docker tags, depending on the context:
+
+- on every branch (including `master`), images have following tags:
+  - the first 8 chars of the targetted commit,
+  - the branch name, with `\` replaced by `-`. For instance the branch `jd/fix/1` will have the `jd-fix-1` tag on DockerHub.
```
Comment (simonvadee) [inline fhir-api/CONTRIBUTING.md:71]: typo, backslash instead of slash

**Review by simonvadee — APPROVED** [review]

File: fhir-api/CONTRIBUTING.md (hunk 1 above)
Comment (Jasopaum) [inline fhir-api/CONTRIBUTING.md:71]: typo: `\` -> `/`

File: fhir-api/CONTRIBUTING.md [hunk 2]
```diff
@@ -59,3 +59,38 @@ Code quality is enforced with `pre-commit` hooks: `black`, `isort`, `flake8`
...
+
+#### 2. Push the tag
+
+        git push --tags api/vX.Y.Z
+
+Providing that the CI workflow is successful (which should always be the case on `master`...), a new image will soon be available on DockerHub with the specified tag.
+
+#### 3. Pull the tagged image
+
+        git pull arkhn/fhir-api:vX.Y.Z
```
Comment (Jasopaum) [inline fhir-api/CONTRIBUTING.md:96]: typo: git pull -> docker pull

**Review by Jasopaum — APPROVED** [review]

File: fhir-api/CONTRIBUTING.md (hunk 2 above)
Comment (vmttn) [reply fhir-api/CONTRIBUTING.md]: lol

**Pull request merged**